core/src/test/java/org/apache/calcite/test/RexProgramTest.java

```diff
@@ -2170,6 +2170,22 @@ private RexNode simplify(RexNode e) {
         is(false));
   }
 
+  @Test public void testIsAlwaysTrueAndFalse() {
+    final RelDataType booleanNullableType =
+        typeFactory.createTypeWithNullability(
+            typeFactory.createSqlType(SqlTypeName.BOOLEAN), true);
+    RexNode node = rexBuilder.makeInputRef(booleanNullableType, 0);
```

Comment:
I am sure it makes no sense to clutter test code with boiler-plate creation of `RelDataType` and `inputrefs`